src/commons/sideContent/content/SideContentCseMachine.tsx

```
@@ -596,10 +542,74 @@ export const SideContentCseMachine = connect(
+  return (
+    <div
+      id="cse-machine-default-text"
+      className={Classes.RUNNING_TEXT}
+      data-testid="cse-machine-default-text"
+    >
+      {isJava ? (
+        <span>
+          <Trans
+            ns="sideContent"
+            i18nKey="cseMachine.csecDescription"
+            components={[<ItalicLink href={Links.textbookChapter3_2} />]}
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)